index.html

```diff
@@ -21,36 +21,54 @@
                
                 <div class="toolbar">
                     <div class="logo"><a href="#">Emmykage</a> </div>
+                  
+                    <ul> <li><a href="#work">Portfolio</a></li>
+                        <li><a href="#about">About</a></li>
+                        <li><a href="#contact">contact</a></li>
+                        <li><a href="#"><img src="image/msessage_Button/Tertiary/Icons/Icon-Mail.svg" alt="mail"></a></li>
+
+                    </ul>
+                    
                         <div class="menu-bar"><img src="Normal_button/Tertiary/Icons/Icon-Menu.png" alt="menu"></div> 
                 </div>
 
             </nav>
         </header>
         <section class="headline">
-            <h1>Hey There.
-                I’m Morris</h1>
-                <h3>I am a software developer</h3>
-                <p>
-                    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-don’t hestiate to contact me.
-                </p>
+            <div class="headline-body">
+                <h1>Hey There.
+                    I’m Morris</h1>
+                    <h3>I am a software developer</h3>
+                    <p>
+                        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+    don’t hestiate to contact me.
+                    </p>
+
+            </div>
+            <div class="headlink">
                 <ul class="social-links"><li>
                     
                    
-                    <li><a href="#"> <img src="Normal_Button/Tertiary/icons/Icon-GitHub.png" alt="git"></a></li>
-                    <li><a href="#"><img src="Normal_Button/Tertiary/icons/Icon-Linkedin.png" alt="linkedin"> </a></li>
-                    <li><a href="#"><img src="Normal_Button/Tertiary/icons/Icon-Angelist.png" alt="angel"> </a></li>
-                    <li><a href="#"> <img src="Normal_Button/Tertiary/icons/Icon-Twitter.png" alt="twitter"></a></li>
-                    <li><a href="#"><img src="Normal_Button/Tertiary/icons/Icon-Medium.png" alt="medium"> </a></li>
+                    <li><a href="#"> <img src="./Normal_Button/Tertiary/icons/Icon-GitHub.png" alt="git"></a></li>
+                    <li><a href="#"><img src="./Normal_Button/Tertiary/icons/Icon-Linkedin.png" alt="linkedin"> </a></li>
+                    <li><a href="#"><img src="./Normal_Button/Tertiary/icons/Icon-Angelist.png" alt="angel"> </a></li>
+                    <li><a href="#"> <img src="./Normal_Button/Tertiary/icons/Icon-Twitter.png" alt="twitter"></a></li>
+                    <li><a href="#"><img src="./Normal_Button/Tertiary/icons/Icon-Medium.png" alt="medium"> </a></li>
```

Comment:
- [ ] Your page social media icons are not showing. As the previous reviewer suggested
`<img src="./Normal_Button/Tertiary/Icons/Icon-GitHub.png"git">`, You missed one of the part. One of the folder names is `Icons` not `icons`. The Changed in the first capital letter for `icons` makes the issue. Please fix it. You can change the folder name or the image path. :+1: 

- [ ] Kindly fix the issue also for the social media icons in the footer. :+1: 

https://github.com/Emmykage/portfolio/blob/6bf27dcf1dccee726260143ac1f406ba2c769c12/index.html#L232-L236

Reply:
the links appears to be perfect and works very well on my device...